src/main/java/org/broadinstitute/consent/http/db/UserPropertyDAO.java

```diff
@@ -17,9 +17,11 @@
 @RegisterRowMapper(UserPropertyMapper.class)
 public interface UserPropertyDAO extends Transactional<UserPropertyDAO> {
 
-    @SqlQuery("SELECT * FROM user_property WHERE userid = :userId")
-    List<UserProperty> findResearcherPropertiesByUser(@Bind("userId") Integer userId);
+    @SqlQuery("SELECT * FROM user_property WHERE userid = :userId AND propertykey IN (<properties>)")
+    List<UserProperty> findResearcherPropertiesByUser(@Bind("userId") Integer userId,
+                                                      @BindList("properties") List<String> properties);
```

Comment:
I really like this approach, it retains the existing outdated data so we don't lose it.

Reply:
thanks !!